audio/src/ContentReactor.Audio/ContentReactor.Audio.Services/AudioService.cs

```diff
@@ -46,39 +47,46 @@ public AudioService(IBlobRepository blobRepository, IAudioTranscriptionService a
             var audioId = Guid.NewGuid().ToString();
 
             // create a blob placeholder (which will not have any contents yet)
-            var blob = BlobRepository.CreatePlaceholderBlob(AudioBlobContainerName, userId, audioId);
+            var blob = BlobRepository.CreatePlaceholderBlob(AudioBlobContainerName, audioId);
 
             // get a SAS token to allow the client to write the blob
-            var writePolicy = new SharedAccessBlobPolicy
+            BlobSasBuilder sasBuilder = new BlobSasBuilder
```

Comment:
nit:  The surrounding code is predominantly using `var` and there's no ambiguity here as to the type being instantiated.   For consistency, I'd recommend using `var` here.

Reply:
Resolved